test/components/primer/open_project/collapsible_section_test.rb

```
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require "components/test_helper"
+
+class PrimerOpenProjectCollapsibleSectionTest < Minitest::Test
+  include Primer::ComponentTestHelpers
+
+  def test_renders
+    render_inline(Primer::OpenProject::CollapsibleSection.new) do |component|
+      component.with_title { "Hello world" }
+      component.with_collapsible_content { "HIDE ME" }
+    end
+
+    assert_selector(".CollapsibleSection")
+    assert_text("Hello world")
+    assert_text("HIDE ME")
+  end
+
+  def test_does_not_render_without_title_and_content
+    render_inline(Primer::OpenProject::CollapsibleSection.new)
+
+    refute_selector ".CollapsibleSection"
```

We're not really consistent in how we make slots required (neither is upstream Primer AFAICT). Should we fail silently and not render (as you've implemented), or should we raise an `ArgumentError` (the behaviour of the `DangerDialog`, for example).

---

I changed it to throw an error. Now it is at least consistent with the `BorderBox::CollapsibleHeader`